documentation/docs/steps/abapEnvironmentPullGitRepo.md

```diff
@@ -24,14 +24,35 @@ abapEnvironmentPullGitRepo script: this
 If you want to provide the host and credentials of the Communication Arrangement directly, the configuration could look as follows:
 
 ```yaml
-
 steps:
   abapEnvironmentPullGitRepo:
     repositoryNames: ['/DMO/GIT_REPOSITORY']
     abapCredentialsId: 'abapCredentialsId'
     host: '1234-abcd-5678-efgh-ijk.abap.eu10.hana.ondemand.com'
 ```
 
+Also you can specify a list of dedicated files containing the repositories to be pulled:
+
+```yaml
+steps:
+  abapEnvironmentPullGitRepo:
+    repositoryNamesFiles: ['repositories.yml']
```

Comment:
abapEnvironmentCloneGitRepo uses the parameter "repositories".

Reply:
changed it to "repositories"